src/js/components/Diagram/stories/typescript/Connections.tsx

```diff
@@ -40,6 +42,6 @@ const Connections = () => {
   );
 };
 
-storiesOf('TypeScript/Diagram', module).add('Connections', () => (
-  <Connections />
-));
+if (!isChromatic()) {
```

Comment:
We can enable chromatic if the story doesn't exist in js

Reply:
Enabled Chromatic testing for `Connections.tsx`